platform-manageability-agent/configs/platform-manageability-agent.yaml

```diff
@@ -0,0 +1,24 @@
+# SPDX-FileCopyrightText: (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+---
+version: v0.1.0
+logLevel: info
+GUID: 'aaaaaaaa-0000-1111-2222-bbbbbbbbcccc'
+manageability:
+  enabled: true
+  serviceURL: infra.test.edgeorch.intel.com:443
+  heartbeatInterval: 10s
+status:
+  endpoint: '/run/platform-manageability-agent/platform-manageability-agent.sock'
```

Comment:
Is this endpoint for sending the agent status to node agent or is it used for a different purpose?

Reply:
To pass the platform-manageability-agent status to the node-agent, the platform-manageability-agent should send the status to the Unix socket created by the node-agent. The node-agent does not query directly to the other agents. Since this is for agent status reporting, it should be configured with the same socket address as in https://github.com/open-edge-platform/edge-node-agents/blob/main/hardware-discovery-agent/config/hd-agent.yaml#L15